packages/svn-extractor/PKGBUILD

```diff
@@ -2,14 +2,14 @@
 # See COPYING for license details.
 
 pkgname=svn-extractor
-pkgver=39.39941be
-pkgrel=2
+pkgver=42.c27831c
+pkgrel=1
 groups=('blackarch' 'blackarch-scanner')
```

Comment:
Please add `blackarch-recon` to the groups.

Reply:
Squashed and pushed @Edu4rdSHL

Reply:
Thank you!